lab_7_llm/main.py

```diff
@@ -32,6 +27,7 @@ class RawDataImporter(AbstractRawDataImporter):
     """
     A class that imports the HuggingFace dataset.
     """
+    _raw_data: DataFrame
```

Comment:
This parameter is already created in the class `AbstractRawDataImporter`

Reply:
Without this line the code cannot pass mypy checks